t/plugin/proxy-mirror.t

```diff
@@ -32,10 +32,11 @@ add_block_preprocessor(sub {
         server_tokens off;
 
         location / {
-            content_by_lua_block {            
+            content_by_lua_block {
                 local core = require("apisix.core")
 
-                local headers_tab = ngx.req.get_headers()               
+                core.log.info("upstream_http_version: ", ngx.req.http_version())
+                local headers_tab = ngx.req.get_headers()
                 for k, v in pairs(headers_tab) do
```

Comment:
I run the tests locally. It seems the order of `headers_tab` is unstable?

Reply:
Thank you very much for your suggestions, I will update later.

Reply:
hi, @spacewander , I tried it locally. The `headers_tab` printed to the log does not need to be sorted. We need to sort the `_M.uri()` function in `t/lib/server.lua`. I think we can solve this by creating a new pr problem. What do you think?